add systemd info and instructions

feinedsquirrel opened the pull request:
In conjunction with the pull request in the codebase. Let me know if any changes should be made. I also fixed the deprecated reference to `ui/dist` and updated it to `ui/build`.

viktorstrate commented:
Looks good, nice explaination of everything. I am wondering if we should make a separate page for systemd configuration.

- It might be a bit hard to navigate the page if there is too much information on the page and you just want to get it up and running as easily as possible.
- Also the "Copy needed files" step is different which also makes the page harder to read and understand.

Simply make a new file called `installation-systemd.md` whith the following header:

```markdown
---
title: Use with systemd
group: Installation
priority: 3
---
```

What do you think?

feinedsquirrel commented:
Yes, I agree. As the steps started growing I had that exact same thought. I'll try to get it done within the next couple of days, and you can double check everything, again.

feinedsquirrel commented:
Alright, I have moved the `systemd` stuff to its own file. The info about the large cache seemed generic enough to any type of installation that I decided to move it out of the `installation-systemd.md` but couldn't decide where best to put it. For now I placed it in the faq.md page, let me know what you think of everything.
I also fixed a minor grammatical error in the original installation-manual.md page.

viktorstrate commented on src/docs/installation-systemd.md:
```diff
@@ -0,0 +1,78 @@
+You can optionally use `systemd` to manage photoview and start the program at boot.
+It also allows the program to run as its own system user, enhancing the security of the process.
+
+To get started, follow [Preparation](), [Download and build Photoview](), [Build the Web user-interface](), and [Build the API back-end]().
```
Should this line be removed? It sounds a lot like the one below :)

```suggestion
```

feinedsquirrel replied:
Oh, ha, yeah. I re-wrote the line, then forgot to remove the original version. Good catch!

viktorstrate replied:
I though that was the case, anyways thank you for helping out!
